### DISCUSSION TITLE
banned from discord?

### DISCUSSION THREAD

**Discussion by christianfurr** [Rapptz/discord.py#9920] [General] [answered]
brother why do i be bannned?? idk what i did. i need some mad help on this error bro and im tryna ask the server but ur boy is banned and i dont see a ban appeal server so ima cooked fr

**Comment by christianfurr** [comment]
`577985634359050251`

my discord id bros

**Comment by Rapptz** [comment, marked as answer]
You got banned for advertising. I've unbanned you.

**Reply by christianfurr** [reply]
Thank you!
